BudgetSquirrel.Api/Services/Implementations/BudgetService.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BudgetSquirrel.Api.ResponseModels;
+using BudgetSquirrel.Api.Services.Interfaces;
+using BudgetSquirrel.Business;
+using BudgetSquirrel.Business.Auth;
+using BudgetSquirrel.Business.BudgetPlanning;
+using BudgetSquirrel.Data.EntityFramework;
+using BudgetSquirrel.Data.EntityFramework.Models;
+using BudgetSquirrel.Data.EntityFramework.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace BudgetSquirrel.Api.Services.Implementations
+{
+  public class BudgetService : IBudgetService
+  {
+    private readonly IAuthService authService;
+    private readonly IAsyncQueryService asyncQueryService;
+    private readonly BudgetSquirrelContext context;
+
+    public BudgetService(IAuthService authService, IAsyncQueryService asyncQueryService)
+    {
+      this.authService = authService;
+      this.asyncQueryService = asyncQueryService;
+    }
```

Comment:
We have 3 readonly properties but are only assigning two.

Reply:
lol obviously we don't need context. I forgot to delete it haha